### ISSUE TITLE
new release time?

### ISSUE THREAD

**Issue by vtolstov** [hashicorp/memberlist#170]
after merging go.mod may be this is time to new release? @dadgar what do you think?

**Comment by dadgar** [comment]
Hey I am not maintaining this repo, pinging @banks

**Comment by vtolstov** [comment]
@banks what do you think?

**Comment by banks** [comment]
We're discussing a release strategy for go-mod era for these libs. Since we don't yet use go mod for any of the upstream projects we manage and don't currently have a separate release process/cycle for these we want to work out what makes the most sense.

The biggest problem in this case is that the repo already included the v0.1.0 tag which was not really part of any proper version/release strategy but was just a marker for the point before we made a major protocol change. Now go mod has come along it becomes the "latest" release which was not really ever the intention.

Creating a tag is trivial, but we tend to put a much higher bar on our releases than just "CI tests pass" which is why we are thinking a bit before we do just tag it.

We may wait until the new additions which are now merged into Consul master, have been through _their_ release process for 1.4.1 which at least includes a bunch of integration and performance testing which gives us a little more confidence that what is master here is good for other consumers too...

Are you blocked in some way by this @vtolstov?
